app/Jobs/FetchCollectionBanner.php

```diff
@@ -41,7 +41,7 @@ public function handle(): void
         );
 
         $this->collection->extra_attributes->set('banner', $banner);
-        $this->collection->extra_attributes->set('banner_updated_at', now()->timestamp);
+        $this->collection->extra_attributes->set('banner_updated_at', now()->toIso8601String());
```

Comment:
you can keep the timestamp, but in that case change the `CollectionDetailData` signature so it matches the actual data being stored

Reply:
advantage of timestamp is that you can change your check in the collection controller from `$bannerUpdatedAt->diffInDays(now()) > 7` to `$bannerUpdatedAt < now()->subDays(7)->timestamp` so you don't have to parse it

Reply:
in that case I'd simply use `Carbon::now()` as that already stores a string representation and will keep it in line with other `something_at` fields we use (e.g. see `Jobs/FetchNativeBalances.php`. In that case also change the other instance where you added `toIso8601String` in a prior PR